### PR TITLE
Improved docs for how to use go templates to traverse json

### PR DESCRIPTION
**Pull request by andersjanmyr**
A new pull request for clarifying how to traverse the JSON. It would have helped me out since I didn't read the Go template docs properly. Use it if you like it.

### CODE REVIEW

File: docs/templates.md [hunk 1]
```diff
@@ -148,6 +148,22 @@ keys = [
 {{end}}
 ```
 
+#### Traversing the Map created from json
```
Comment (bacongobbler) [inline docs/templates.md]: Let's simplify this to "Advanced Map Traversals" for consistency with other titles.

File: docs/templates.md [hunk 2]
```diff
@@ -148,6 +148,22 @@ keys = [
...
+
+Once you have parsed the JSON, it is possible to traverse it with normal Go
+template functions such as `index`.
+
+The structure {"animals": [{"type": "dog", "name": "Fido"}, {"type": "cat", "name": "Misse"}]}` can be traversed like this:
```
Comment (bacongobbler) [inline docs/templates.md]: missing a backtick

**Comment by andersjanmyr** [comment]
Alright I improved the docs as you suggested and also split the JSON over more lines for better readability.

**Comment by bacongobbler** [comment]
lgtm

**Pull request merged**